docker/docker-compose.yml

```diff
@@ -0,0 +1,51 @@
+version: '3'
```

Comment:
Why not `3.7`? [docker-compose version and documentation](https://docs.docker.com/compose/compose-file/)